src/main/java/com/juniorstart/juniorstart/email/EmailMassageTemplate.java

```diff
@@ -0,0 +1,53 @@
+package com.juniorstart.juniorstart.email;
+
+import java.io.Serializable;
+import java.time.LocalDate;
+
+/** Singleton class with static methods for build email 12-09-2020.
```

Comment:
Nie wiem czy te daty są tutaj potrzebne.

Reply:
zależy od koncepcji. jak robię kurs guru to on ma atuomat co mu od razu wpisuje date.